.env.sample

```diff
@@ -1,8 +0,0 @@
-REACT_APP_ENDPOINT=http://localhost:3000
-REACT_APP_CLIENT_ID=0oavsomz01GzuGikY4x6
-REACT_APP_OKTA_ISSUER_URI=https://auth.lambdalabs.dev/oauth2/default
-REACT_APP_API_URI=http://localhost:8000
-REACT_APP_DS_API	https://a-ds.storysquad.dev/	
-REACT_APP_DS_TOKEN	xD5w6CTfQSh6qs4xviIV6viQomie3ePjQQeDQjsr	
-REACT_APP_S3_KEY	AKIASVA5GJL5XYJ7V75C	
-REACT_APP_S3_SECRET_KEY	MY4fQSYV2URovujDtF3IQgDoceJMmkA7qm+5bTpT
```

Comment:
.env.sample was pulled down from the repo the last time I pulled from main - in order for it not to be pushed up again I removed the contents of this file.